src/CSSFramework/BootstrapCSSFramework.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace TheWebmen\ElementalGrid\CSSFramework;
+
+use DNADesign\Elemental\Models\BaseElement;
+
+final class BootstrapCSSFramework implements CSSFrameworkInterface
+{
+    /**
+     * @var BaseElement
+     */
+    private $baseElement;
+
+    private const COLUMN_CLASSNAME = 'col';
+
+    private const ROW_CLASSNAME = 'row';
+
+    private const CONTAINER_CLASSNAME = 'container';
+
+    private const FLUID_CONTAINER_CLASSNAME = 'container-fluid';
+
+    /**
+     * @param BaseElement $baseElement
+     */
+    public function __construct($baseElement)
+    {
+        $this->baseElement = $baseElement;
+    }
+
+    /**
+     * @return string
+     */
+    public function getRowClasses()
+    {
+        return self::ROW_CLASSNAME;
+    }
+
+    /**
+     * @return string
+     */
+    public function getColumnClasses()
+    {
+        $sizeClasses = $this->getSizeClasses();
+        $offsetClasses = $this->getOffsetClasses();
+        $visibilityClasses = $this->getVisibilityClasses();
+
+        $classes = array_merge($sizeClasses, $offsetClasses, $visibilityClasses);
+
+        return implode(' ', $classes);
+    }
+
+    /**
+     * @return array
+     */
+    private function getVisibilityClasses()
+    {
+        $classes = [];
+
+        if ($this->baseElement->VisibilityXS === 'hidden') {
+            array_push($classes, 'd-none d-sm-block');
```

Comment:
Not sure about the usage of `array_push` when adding a single value to the array. Even the official [documentation](http://docs.php.net/array_push) advices against that.

> Note: If you use array_push() to add one element to the array, it's better to use $array[] = because in that way there is no overhead of calling a function.

Reply:
As discussed with Erik on slack, will not be changed, due to the fact frontenders also copy paste this method a lot, this makes it easier to leave the code intact for a frontender and to also add multiple items to the array.